apps/core/tests/.env.example

```diff
@@ -0,0 +1,2 @@
+# Environment variables for tests
+PLAYWRIGHT_TEST_BASE_URL='https://catalyst-demo.site'
```

Comment:
Now that I think about it, let's add remove this for now as we probably need to add [some documentation](https://github.com/bigcommerce/catalyst/blob/main/docs/environment-variables.md) around it.

Reply:
👍 I have it documented in `docs/monorepo.md` but this might also be another placeholder to add it as it's an environment variable.